src/components/interfaces.ts

```diff
@@ -8,33 +8,39 @@ export interface ComponentDefinition {
   version?: string;
   /** @deprecated */
   description?: string;
+  systemTag?: Array<string>;
   properties: ComponentProperty[];
   regions: ComponentRegion[];
   functions: ComponentFunction[];
   events: EventHandler[];
 }
 
-export interface ComponentProperty {
+interface Taggable {
+  analyticsTag?: string;
```

Comment:
Minor: `analyticsTag` can stay in ComponentProperty. When used in a ComponentRegion it will be ignored.

Reply:
Well, I see `EventHandler` uses even less of these properties, so you're just favoring the code organization and being permissive with inheritance here. No big concern.